### PR TITLE
Change function signature from rest.API to rest.Send.

### PR DESCRIPTION
**Pull request by janczer**
According to #13.

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/rest?pullRequest=19) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/rest?pullRequest=19) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/rest?pullRequest=19) it.</sub>

**Comment by thinkingserious** [comment]
HI @janczer,

So that we don't have a breaking change, can you please implement `rest.API` as a pass through function that simply calls `rest.Send`?

With Best Regards,

Elmer

**Comment by janczer** [comment]
Hi @thinkingserious 

Sure. I added it and smash to the one commit.

File: rest_test.go [hunk 1]
```diff
@@ -95,7 +95,7 @@ func TestRest(t *testing.T) {
 		Headers:     Headers,
 		QueryParams: queryParams,
 	}
-	response, e := API(request)
```
Comment (thinkingserious) [inline rest_test.go]: Please add these tests back in, in addition to the new ones you created for `Send`. Thanks!

Comment (janczer) [reply rest_test.go]: I add function `testingAPI`, and send to this function like parameter functions `Send` and `API` because code the same.

**Comment by thinkingserious** [comment]
My apologies @janczer,

Could you please update the merge conflicts?

Thanks!

With Best Regards,

Elmer

**Comment by janczer** [comment]
@thinkingserious 

I rebased my branche and merge conflicts.

Thanks!

**Comment by thinkingserious** [comment]
Hello @janczer,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/announcing-hacktoberfest-2017">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Comment by jackgardner** [comment]
@janczer @thinkingserious 

I think you are also missing the API() passthrough on the Client struct, or sendgrid-go also needs a PR: 

https://github.com/sendgrid/sendgrid-go/blob/238cd04b958416fad59abdac707079c91d080bc9/sendgrid.go#L54

**Comment by thinkingserious** [comment]
Yes, sendgrid-go will need an updated PR. That would be a most excellent [hacktoberfest](https://sendgrid.com/blog/announcing-hacktoberfest-2017/) contribution ;)

**Comment by jerbob92** [comment]
Honestly, I do not appreciate breaking the package API. 
I created a MR (#68) to also add fallback functionality for the API() function in the Client.

**Comment by thinkingserious** [comment]
Thanks for catching this @jerbob92!

**Pull request merged**